### PR TITLE
Feature/more results extend

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-0.22.0 / 2017-12-12
+0.23.0 / 2017-12-14
```
Comment (ajthomascouk) [inline CHANGELOG.md]: Why are we skipping `0.22.0` ?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "connecting-to-services",
-  "version": "0.22.0",
+  "version": "0.23.0",
```
Comment (ajthomascouk) [inline package.json]: Same question as above.

File: CHANGELOG.md (hunk 1 above)
Comment (ajthomascouk) [inline CHANGELOG.md]: why are we skipping `0.22.0` ?

File: package.json (hunk 2 above)
Comment (ajthomascouk) [inline package.json]: Same question as above.

File: CHANGELOG.md (hunk 1 above)
Comment (st3v3nhunt) [reply CHANGELOG.md]: It has already been created.

Comment (ajthomascouk) [reply CHANGELOG.md]: aah `0.22.0` was `Pre-release` I thought it was just `Draft`. All good then.

**Pull request merged**